### PR TITLE
Adds Robotic Storage to Central Command and Reduces CC Toxicity

### PR DESCRIPTION
**Pull request by LightFire53**
## What Does This PR Do
 Adds robotic storage for ERT Cyborgs with the ERT cryodorms.

Also changes the instances of `/turf/simulated/floor/plasteel/dark/nitrogen` to be `/turf/simulated/floor/plasteel/dark` so CC personnel stop being poisoned by the atmosphere. <strike>This is a Certified™ S34N moment.</strike>

## Why It's Good For The Game
 Sleepy borgs want to go to bed. Steve wants to stop suffocating.

## Images of changes

image

## Changelog
:cl:
add: Added robotic storage units to CC Cryodorms
add: Adds an offstation type of robotic storage unit that won't announce when robots enter cryosleep.
fix: Central Command no longer has a toxic atmosphere in places. Note: Personnel may remain toxic.
/:cl:

### CODE REVIEW

**Comment by S34NW** [comment]
https://youtu.be/sTMgX1PDGAE

**Comment by Contrabang** [comment]
Just wondering, are these robotic storages silent like the other cryo storages? Like the normal ones at CC wouldnt say that the ERT has cryod, but these robotic ones might.

**Comment by LightFire53** [comment]
> Just wondering, are these robotic storages silent like the other cryo storages? Like the normal ones at CC wouldnt say that the ERT has cryod, but these robotic ones might.

Not in the latest commit now. Thanks for mentioning!

File: code/game/machinery/cryopod.dm [hunk 1]
```diff
@@ -702,6 +702,10 @@
 	allow_occupant_types = list(/mob/living/silicon/robot)
 	disallow_occupant_types = list(/mob/living/silicon/robot/drone)
 
+/obj/machinery/cryopod/robot/offstation
+	//For offstation cyborgs
+	silent = TRUE
+
```
Comment (Miraviel) [inline code/game/machinery/cryopod.dm:708]: When you cryo as a cyborg, it does not show up on the local Robotic Storage Console (neither on the on-station one):

image

...

It shows up in the Cryogenic Oversight Console instead:

image

**Review by Miraviel — CHANGES_REQUESTED** [review]
Tested it, I could not find anything else problematic other than the robotic console. In case anyone would like to test these, I've done so:

- If I cryo in CC, will my character show up in the cryostorage on the station? - no
- Is the cryogenic console on the wall linked to these cryopods? - yes
- Is the on-station console linked to these cryopods in any way? - no
- Is the area lit? - yes
- Do airlocks need any kind of access? - no
- Do people properly despawn? - yes

Dunno what else could be tested, it just works.

**Review by S34NW — APPROVED** [review]
map is fine

**Review by Miraviel — APPROVED** [review]
Apparently, the cryo storage thing is a code-related issue, I will try to fix that. Map is neat!

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**